karma.conf.js

```diff
@@ -53,7 +53,16 @@ module.exports = function(config) {
 
       "website/public/js/filters/filters.js",
 
-      "website/public/js/directives/directives.js",
```

Comment:
That's a good point. I wonder if there's a way in the manifest.json to easily do `.../*.js` syntax?